source/Applications/Advanced/HandEyeCalibration/PoseConversions/PoseConversions.cpp

```diff
@@ -34,30 +34,25 @@ struct RollPitchYaw
     Eigen::Array3d rollPitchYaw;
 };
 
-struct Representations
-{
-    Eigen::AngleAxisd axisAngle;
-    Eigen::Vector3d rotationVector;
-    Eigen::Quaterniond quaternion;
-    std::array<RollPitchYaw, nofRotationConventions> rotations;
-};
-
 Eigen::Affine3d getTransformationMatrixFromYAML(const std::string &path);
 void saveTransformationMatrixToYAML(const Eigen::Affine3d &, const std::string &path);
 cv::Mat eigenToCv(const Eigen::MatrixXd &);
 Eigen::MatrixXd cvToEigen(const cv::Mat &);
 Eigen::Array3d rotationMatrixToRollPitchYaw(const Eigen::Matrix3d &rotationMatrix, const RotationConvention &rotation);
 Eigen::Matrix3d rollPitchYawToRotationMatrix(const Eigen::Array3d &rollPitchYaw, const RotationConvention &rotation);
+Eigen::Vector3d rotation_matrix_to_rotation_vector(const Eigen::Matrix3d &rotationMatrixs);
```

Comment:
Use camelCase for C++

Reply:
rotationMatrix**s** - remove 's' at the end to get rotationMatrix

Reply:
just installed https://marketplace.visualstudio.com/items?itemName=munyabe.CaseConverter